tinygrad/codegen/uops.py

```diff
@@ -41,10 +41,10 @@ def wfxn(*args):
   TernaryOps.WHERE: lambda x,y,z: y if x else z}
 
 def exec_alu(arg, dtype, p):
-  return python_alu[arg](*p)
-  #if not dtypes.is_int(dtype): return ret
-  #adjusted = 0 if dtypes.is_unsigned(dtype) else 2 ** (dtype.itemsize * 8 - 1)
-  #return (ret + adjusted) % 2 ** (dtype.itemsize * 8) - adjusted
+  ret = python_alu[arg](*p)
+  if not dtypes.is_int(dtype): return ret
```

Comment:
I suggest (once) constructing a dictionary of functions to apply the truncation for dtype. Like Dict[dtype, Callable]